heron/storm/src/java/org/apache/storm/task/TopologyContext.java

```diff
@@ -186,6 +190,14 @@ public int getThisTaskIndex() {
         return getSources(getThisComponentId());
     }
   */
+  public Set<GlobalStreamId> getThisSourceIds() {
```

Comment:
I'm not sure if this is the correct way. Right now Heron doesn't have the Grouping class, so I created the method to get all source streams. It's used by WindowedBoltExecutor.